packages/zoe/src/contractSupport/interest.js

```diff
@@ -27,10 +27,10 @@ const calculateRelativeCompounding = (
 
 /**
  *
- * @param {Amount<NatValue>} debtSnapshot
+ * @param {Amount<'nat'>} debtSnapshot
```

Comment:
All the methods in this file are only used within Vaults. Let's move it there for a start.

The follow-on question is whether it should be merged in with the new `interest.js`.  I guess these only apply to the way that `Vaults` aggregate interest charges at the `VaultManager` level, so that would argue that it makes sense to leave them separate.

Would you mind moving it and adding comments about how special-purpose this is?

Reply:
>  I guess these only apply to the way that Vaults aggregate interest charges at the VaultManager level, so that would argue that it makes sense to leave them separate.

The refactoring was prompted by working with @dckc on getRUN . It's not a vault and needs to charge interest and is in progress https://github.com/Agoric/agoric-sdk/pull/4731 

So I'll move it from Zoe's `interest.js` to Run-protocol's.